src/modlunky2/assets/assets.py

```diff
@@ -182,7 +182,7 @@ def extract(
 
             except Exception:  # pylint: disable=broad-except
                 logger.exception("Failed compression")
-                return None
+                return
```

Comment:
Why does this one remove the explicit None?

Reply:
This function never returns a value, and the returned value is never used. If there's no exception, it just implicitly returns `None`.

I think plain `return` makes more sense here